npm/ux/resultsTable.tsx

```diff
@@ -194,9 +204,9 @@ export function ResultsTable(props: {
   }
 
   function rowClicked(rowId: string) {
-    if (selectedRow === rowId && props.ensureSelected) return;
+    if (props.selectedRow === rowId && props.ensureSelected) return;
 
-    const newSelectedRow = selectedRow === rowId ? "" : rowId;
+    const newSelectedRow = props.selectedRow === rowId ? "" : rowId;
     setSelectedRow(newSelectedRow);
     props.onRowSelected(newSelectedRow);
```

Comment:
Being that setSelectedRow now just calls `props.setSelectedRow()`, do we still need to call `props.onRowSelected()` to notify the parent if we set it by calling the parent?

Reply:
Yep! We could keep just one callback out of 2.